### PR TITLE
audacity2.4.2: Add version 2.4.2

### PR DESCRIPTION
**Pull request by Ganesha2282882**
I have added Audacity here. FOSSHub doesn't provide direct downloads (they expire) so I put it on archive.org.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/audacity242.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "version": "2.4.2",
+    "description": "Audacity is an easy-to-use, multi-track audio editor and recorder for Windows, macOS, GNU/Linux and other operating systems.",
```
Comment (tech189) [inline bucket/audacity242.json:3]:
```suggestion
    "description": "Audacity is an easy-to-use, multi-track audio editor and recorder for Windows, macOS, GNU/Linux and other operating systems.",
    "homepage": "https://www.audacityteam.org",
    "license": "GPL-2.0-or-later",
```
Add the homepage and license to pass checks

File: bucket/audacity242.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "version": "2.4.2",
+    "description": "Audacity is an easy-to-use, multi-track audio editor and recorder for Windows, macOS, GNU/Linux and other operating systems.",
+    "url": "https://archive.org/download/audacity-2.4.2_202205/audacity-2.4.2.zip",
+    "hash": "0c14f7c6850c93b9dacc14fe66876b8dc3397d92dbd849898783a21bad1fff55",
```
Comment (tech189) [inline bucket/audacity242.json:5]: https://github.com/ScoopInstaller/Extras/blob/a0dddff0dee46b9b668f5aec0e53ea029c17d783/bucket/audacity.json#L7

Good that the hash matches the old manifest above, but maybe we should add the binary to the [Scoop Binary archive](https://github.com/ScoopInstaller/Binary) instead?

**Comment by rashil2000** [comment]
What's the significance of version 2.4.2?

**Comment by Ganesha2282882** [comment]
No tracking

File: bucket/audacity242.json (hunk 1 above)
Comment (Ganesha2282882) [reply bucket/audacity242.json:3]: I added the license, it might have conflicts.

File: bucket/audacity242.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "version": "2.4.2",
+    "description": "Audacity is an easy-to-use, multi-track audio editor and recorder for Windows, macOS, GNU/Linux and other operating systems.",
+    "license": "GPL-2.0-or-later",
+    "url": "https://archive.org/download/audacity-2.4.2_202205/audacity-2.4.2.zip",
+    "hash": "0c14f7c6850c93b9dacc14fe66876b8dc3397d92dbd849898783a21bad1fff55",
+    "bin": "audacity.exe",
```
Comment (rashil2000) [inline bucket/audacity242.json:7]: CLI args?

File: bucket/audacity242.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+    "version": "2.4.2",
+    "description": "Audacity is an easy-to-use, multi-track audio editor and recorder for Windows, macOS, GNU/Linux and other operating systems.",
+    "license": "GPL-2.0-or-later",
+    "url": "https://archive.org/download/audacity-2.4.2_202205/audacity-2.4.2.zip",
+    "hash": "0c14f7c6850c93b9dacc14fe66876b8dc3397d92dbd849898783a21bad1fff55",
+    "bin": "audacity.exe",
+    "shortcuts": [["audacity.exe", "Audacity"]]
```
Comment (rashil2000) [inline bucket/audacity242.json:8]:
```suggestion
    "shortcuts": [
        [
            "audacity.exe",
            "Audacity"
        ]
    ]
```

**Comment by rashil2000** [comment]
Add homepage, and a comment on top saying what is special about this version (no tracking), similar to the comment here: https://github.com/ScoopInstaller/Main/blob/master/bucket/gpg.json

File: bucket/audacity242.json (hunk 3 above)
Comment (issaclin32) [reply bucket/audacity242.json:7]: > https://forum.audacityteam.org/viewtopic.php?t=77335
Audacity only has limited command-line support. The only current command-line functions are that you can open audio files or Audacity project files (including multiple files) by passing them as arguments, 
for example:
audacity file1.aiff file2.wav file3.ogg

> The following options are also supported:
-help
display a brief list of command line options
-version
display the Audacity version number
-test
run "Benchmark" diagnostics tests
-blocksize
set the Audacity blocksize for writing files to disk of nnn bytes

**Comment by issaclin32** [comment]
I updated the manifest because the author did not respond for a while.

**Review by rashil2000 — APPROVED** [review]

**Comment by rashil2000** [comment]
Should we rename it to audacity2.4.2.json?

**Comment by issaclin32** [comment]
> Should we rename it to audacity2.4.2.json?

Sure.

**Pull request merged**